.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-- package-ecosystem: npm
+- package-ecosystem: yarn
```

Comment:
I saw "Dependabot broken.", but master build was successful. Could you please share a bit more about the major trigger of this PR?

Reply:
Of course. Dependabot was reporting that there were vulnerabilities in some packages. But it could not generate automatic PRs for this. I found that the reason most probably was, that there were package-locks in those packages, that were broken, e. g. the package-lock of integration did not contain the local packages the integration-test package relies on. So I ended up fixing this manually with an ugly workaround (Remove the local dependencies first, run npm install, add dependencies again). Obviously these packages cannot be handled by dependabot.
Another problem is, that dependabot somehow succeeded in fixing some dependencies, but not all - the update check crashes. I believe that this is has the same root cause.

Reply:
I now also added an ADR, to reflect this.